stdlib/array.gr

```diff
@@ -910,6 +913,12 @@ provide let slice = (startIndex, endIndex, array) => {
  * Ordering is calculated using a comparator function which takes two array elements and must return 0 if both are equal, a positive number if the first is greater, and a negative number if the first is smaller.
  * @param comp: The comparator function used to indicate sort order
  * @param array: The array to be sorted
+ *
+ * @example
+ * let arr = [>3,2,1]
+ * Array.sort((a, b) => a - b, arr)
+ * assert arr == [>1,2,3]
```

Comment:
```suggestion
 * let array = [> 3, 2, 1]
 * Array.sort(compare, array)
 * assert array == [> 1, 2, 3]
```

Reply:
This could be a good time to have one example using `compare` and another using a custom comparator.